pkg/rancher-components/src/components/Card/Card.vue

```diff
@@ -1,8 +1,9 @@
 <script lang="ts">
 import { defineComponent, PropType } from 'vue';
-import { createFocusTrap, FocusTrap } from 'focus-trap';
+import { basicSetupFocusTrap } from '@shell/composables/focusTrap';
```

Comment:
It's a common Vue convention for composable functions to start with `use`. I think that `useFocusTrap` would be a more idiomatic name for this function.

Reply:
updated function names in focus-trap composable to start with `use`